Dockerfile

```diff
@@ -24,7 +18,7 @@ COPY . .
 
 RUN go build -mod vendor -a -ldflags "${LDFLAGS}" -o manager
 
-FROM $BASEIMAGE
+FROM gcr.io/distroless/static-debian12@sha256:8dd8d3ca2cf283383304fd45a5c9c74d5f2cd9da8d3b077d720e264880077c65
```

Comment:
do we not need to keep this so users can override the base image they want to use?

Reply:
not sure if anyone is actually using it like that but this decreases our reproducibility and security since we can't pin (or we can pin but dependabot won't update)

Reply:
if anyone is using it like that they can still do with buildkit source policies, which is a better way to handle these https://docs.docker.com/build/building/variables/#experimental_buildkit_source_policy